pyproject.toml

```diff
@@ -42,4 +42,43 @@ ignore_missing_imports = true
 [tool.isort]
 quiet = true
 line_length = 100
-skip_glob = ["venvs", "build"]
+skip_glob = [".venv", "build"]
+
+[tool.poetry]
```

Comment:
I think this is missing quite a few things from the `setup.py` including one of the most important ones, the entrypoint which makes it a pytest plugin.

Reply:
Will add the entrypoint. Did you find anything else?